src/backend/access/gin/ginxlog.c

```diff
@@ -408,7 +408,6 @@ ginRedoSplit(XLogReaderState *record)
 	bool		isLeaf = (data->flags & GIN_INSERT_ISLEAF) != 0;
 	bool		isRoot = (data->flags & GIN_SPLIT_ROOT) != 0;
 	XLogRedoAction action;
-
```

Comment:
Let's not remove this newline. We have added the `XLogRedoAction action;' line above, but the newline is from upstream. `pgindent` would put it back anyway.